Makefile

```diff
@@ -63,7 +63,8 @@ testlocal: generate test
 swarmtest:
 	# Setup a local schedule against local workers (--strategy local.local)
 	# Then run (with go run) scootapi run_smoke_test with 10 jobs, wait 1m
-	go run ./binaries/setup-cloud-scoot/main.go --strategy local.local run go run ./binaries/scootapi/main.go run_smoke_test --num_jobs 10 --timeout 1m
+	go run ./binaries/setup-cloud-scoot/main.go --strategy local.local --workers '--bundlestore $TMPDIR/bundlestore' run \
```

Comment:
I don't want to have to pass this every time I run setup-cloud-scoot; can we extend setup-cloud-scoot so it also sets up bundlestore?